src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/paymentsreporting/wfbrokersfetch/OrganizationsBrokeredRetrieveWFImpl.java

```diff
@@ -0,0 +1,65 @@
+package it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting.wfbrokersfetch;
+
+import io.temporal.spring.boot.WorkflowImpl;
+import it.gov.pagopa.payhub.activities.activity.organization.BrokersRetrieverActivity;
+import it.gov.pagopa.payhub.activities.activity.organization.OrganizationBrokeredRetrieverActivity;
+import it.gov.pagopa.pu.organization.dto.generated.Broker;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting.OrganizationPaymentsReportingPagoPaFetchWFClient;
+import it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting.config.OrganizationsBrokeredRetrieveWFConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+
+import java.util.List;
+
+@Slf4j
+@WorkflowImpl(taskQueues = {OrganizationsBrokeredRetrieveWFImpl.TASK_QUEUE_ORGANIZATIONS_BROKERED_RETRIEVE})
+public class OrganizationsBrokeredRetrieveWFImpl implements OrganizationsBrokeredRetrieveWF, ApplicationContextAware {
+  public static final String TASK_QUEUE_ORGANIZATIONS_BROKERED_RETRIEVE = "OrganizationsBrokeredRetrieveWF";
+
+  private final OrganizationPaymentsReportingPagoPaFetchWFClient organizationPaymentsReportingPagoPaFetchWFClient;
+
+  private BrokersRetrieverActivity brokersRetrieverActivity;
+  private OrganizationBrokeredRetrieverActivity organizationBrokeredRetrieverActivity;
+
+  public OrganizationsBrokeredRetrieveWFImpl(OrganizationPaymentsReportingPagoPaFetchWFClient organizationPaymentsReportingPagoPaFetchWFClient) {
+    this.organizationPaymentsReportingPagoPaFetchWFClient = organizationPaymentsReportingPagoPaFetchWFClient;
+  }
```

Comment:
This is not a real Spring bean, this will not work, you have to retrieve it using the ApplicationContext in setApplicationContext method